packages/app/src/specs/banners/TrackedWarning.vue

```diff
@@ -0,0 +1,62 @@
+<template>
```

Comment:
Are we not able to use the existing `TrackedBanner` and pass a status of `warning` to get this same behavior?

Reply:
☝️ bump @jordanpowell88 . Seems like this is the exact same as `TrackedBanner` minus the "shown" event being recorded. Couldn't we just add a prop to TrackedBanner to disable those?

Reply:
@mike-plummer I suggested to Jordan to create a new component for this based on TrackedBanner, but looking at it more, I think you are correct. @jordanpowell88 Can you implement Mike's suggestion.  If this does not make the release, then it is fine.

Reply:
Just pushed changes which moves this logic to the `TrackedBanner` @mike-plummer @warrensplayer